src/ruby/ext/grpc/rb_channel.c

```diff
@@ -203,7 +203,10 @@ static VALUE grpc_rb_channel_create_call(VALUE self, VALUE cqueue, VALUE method,
   grpc_channel *ch = NULL;
   grpc_completion_queue *cq = NULL;
   char *method_chars = StringValueCStr(method);
-  char *host_chars = StringValueCStr(host);
+  char *host_chars = NULL;
+  if (host != Qnil) {
```

Comment:
- I opted for flexibility - still allowing the host can still be set, though none of the library code does so at the moment.   This should make it possible to support different hosts later with just ruby changes.